ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/client/auth/JwtSecretKeyLoader.java

```diff
@@ -59,6 +59,7 @@ private SecretKeySpec generateNewSecret() {
   private void writeGeneratedKeyToFile(final Key key) {
     final Path generatedKeyFilePath = beaconDataDirectory.resolve(JWT_SECRET_FILE_NAME);
     try {
+      Files.createDirectories(generatedKeyFilePath.getParent());
```

Comment:
hmm, why wouldn't these directories already exist at this point? It would be under the data dir which goes through a directory creation process elsewhere.

Reply:
Acceptance test starts it in different order